Dockerfile

```diff
@@ -1,39 +1,29 @@
-FROM ruby:2.4.1
+FROM ruby:2.4.4-alpine
```

Comment:
Why did you choose to go with 2.4.4 instead of 2.5.1?

Reply:
Just did this a while back.